### ISSUE TITLE
Zoom Breakout Rooms in Web Client

### ISSUE THREAD

**Issue by jnfoster** [clowdr-app/clowdr-web-app#226]
Zoom breakout rooms do not work for folks accessing the call using the web client within Clowdr...

**Comment by bcpierce00** [comment]
Very sad.  Unlikely to get fixed this week, unfortunately.  People should just use Clowdr rooms for breakouts (and I am happy to help set that up.)

**Comment by sweirich** [comment]
Nate, thanks for letting us know. Sorry it didn't work out.

**Comment by jnfoster** [comment]
No worries. Our fault for not test driving this carefully beforehand. And it was fine because there were only ~8 folks on the Web client, so they just stayed in the main room...

(We didn't use Clowdr because we were familiar with the one-click randomization that Zoom supports...)

**Comment by bcpierce00** [comment]
Yes, clowdr only supports randomization through participants manually rolling n-sided dice.  (Which I think is not that bad a system, actually...)

**Comment by jnfoster** [comment]
Yep. I remember you mentioned that during training :-)

**Comment by jon-bell** [comment]
According to these docs, the zoom web client should support breakout rooms, but only on Chrome (it doesn't support audio on other browsers anyway) - https://support.zoom.us/hc/en-us/articles/115005769646 - hopefully it should not be that hard for us to fix (as long as the zoom client supports it)

**Comment by EdNutting** [comment]
This issue relates to the old UI but has some relevance for the Zoom integration in the new UI so leaving open for now.
